Gemfile

```diff
@@ -35,6 +35,7 @@ group :development, :test do
   gem 'pry'
   gem 'pry-byebug'
   gem 'pry-rails'
+  gem 'rb-readline'
```

Comment:
added to prevent error do to non-existing native readline dependency, Eric and I both experienced this..